dxr/filters.py

```diff
@@ -233,4 +235,4 @@ def _should_be_highlit(self, entity):
         """
         return ((not self._term['qualified'] and
                  super(QualifiedNameFilterBase, self)._should_be_highlit(entity))
-                or entity['qualname'] == self._term['arg'])
+                or is_in(self._term['arg'], entity['qualname']))
```

Comment:
I thought about doing a prefix match on the existing data as well, but converting direct searches to "any old prefix" logic starts a whole rabbit trail of design that I'd rather do another time. :-)